docs/source/guide/qse-1-intro.md

```diff
@@ -0,0 +1,260 @@
+        logical_0_state[int(z, 2)] = 1 / 4
+    for z in [
+        "11011",
+        "00110",
+        "11000",
+        "11101",
+        "00011",
+        "11110",
+        "01111",
+        "10001",
+        "01100",
+        "10111",
+    ]:
+        logical_0_state[int(z, 2)] = -1 / 4
+
+    logical_1_state = np.zeros(32)
+    for z in ["11111", "01101", "10110", "01011", "10101", "11010"]:
+        logical_1_state[int(z, 2)] = 1 / 4
+    for z in [
+        "00100",
+        "11001",
+        "00111",
+        "00010",
+        "11100",
+        "00001",
+        "10000",
+        "01110",
+        "10011",
+        "01000",
+    ]:
+        logical_1_state[int(z, 2)] = -1 / 4
+
+    # Fill up the rest of the matrix with orthonormal vectors
+    matrix = gram_schmidt([logical_0_state, logical_1_state])
+    circuit = cirq.Circuit()
+    g = cirq.MatrixGate(matrix)
+    qubits = cirq.LineQubit.range(5)
+    circuit.append(g(*qubits))
+
+    return circuit
+
+def main():
+  circuit = prepare_logical_0_state_for_5_1_3_code()
+```
+
+## 2. Define an executor
+We define an [executor](executors.md) function which inputs a circuit and returns a `QuantumResult`. Here for sake of example we use a simulator that adds single-qubit depolarizing noise after each moment and returns the final density matrix.
+
+```
+Import cirq
+from mitiq.interface import convert_to_mitiq
+from mitiq.interface.mitiq_cirq import compute_density_matrix
+
+
+def execute_with_depolarized_noise(circuit: QPROGRAM) -> np.ndarray:
+    return compute_density_matrix(
+        convert_to_mitiq(circuit)[0],
+        noise_model=cirq.depolarize,
+        noise_level=(0.01,),
+    )
+
+def main():
+  circuit = prepare_logical_0_state_for_5_1_3_code()
+  executor = execute_with_depolarized_noise
+```
+
+## 3. Observable
+We define an observable in the code subspace: $O = ZZZZZ$. As an example, assume that we wish to compute the expectation value $Tr[\rho O]$ of the observable O. 
+
+```
+def get_observable_in_code_space(observable: List[cirq.PauliString]):
+    FIVE_I = PauliString("IIIII")
+    projector_onto_code_space = [
+        PauliString("XZZXI"),
+        PauliString("IXZZX"),
+        PauliString("XIXZZ"),
+        PauliString("ZXIXZ"),
+    ]
+
+    observable_in_code_space = Observable(FIVE_I)
+    all_paulis = projector_onto_code_space + [observable]
+    for g in all_paulis:
+        observable_in_code_space *= 0.5 * Observable(FIVE_I, g)
+    return observable_in_code_space
+
+def main():
+  circuit = prepare_logical_0_state_for_5_1_3_code()
+  executor = execute_with_depolarized_noise
+  observable = get_observable_in_code_space(PauliString("ZZZZZ"))
+```
+
+## 4. Check Operators and Code Hamiltonian
+We then assign the check operators as a list of pauli strings, and the code Hamiltonian as an Observable for the \[\[5,1,3\]\] code. The check operators of the \[\[5,1,3\]\] code are simply the expansion of the code’s 4 generators: $\[XZZXI, IXZZX, XIXZZ, ZXIXZ\]$
```

Comment:
The equation was not rendering correctly. I would've thought you could do this as well, but I guess not 🤷🏻.
```suggestion
We then assign the check operators as a list of pauli strings, and the code Hamiltonian as an Observable for the \[\[5,1,3\]\] code. The check operators of the \[\[5,1,3\]\] code are simply the expansion of the code’s 4 generators: $[XZZXI, IXZZX, XIXZZ, ZXIXZ]$
```